### PR TITLE
Upgrading to Elasticsearch 6.5 | 7.2 New | LRDOCS-5911

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-5911, I stole the steps from the 7.0 article on upgrading from 2.x > 6.x, but removed the need to install a new connector, since you'll just use the default. Let me know if it needs more work.

### CODE REVIEW

**Comment by lipusz** [comment]
**Started reviewing...** :mag::nerd_face:

File: deployment/articles/03-installing-a-search-engine/02-elasticsearch/08-upgrading-to-elasticsearch_65.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Upgrading to Elasticsearch 6.5
+
+Elasticsearch 6.5.x is the default, most up-to-date supported version of
+Elasticsearch for @product-ver@. If you're upgrading @product@ and still running
+Elasticsearch 6.1, it's time to upgrade your Elasticsearch servers,too. If
```
Comment (lipusz) [inline deployment/articles/03-installing-a-search-engine/02-elasticsearch/08-upgrading-to-elasticsearch_65.markdown]: Add: a space before "too". :)

**Comment by lipusz** [comment]
Looks good to me, thanks Russ! :+1:

**Comment by rbohl** [comment]
Forwarded: https://github.com/sez11a/liferay-docs/pull/4308 :+1:

**Pull request closed without merging**